doc/JSON_INFO.md

```diff
@@ -140,10 +140,10 @@ A few properties are applicable to most if not all json files and do not need to
 | Identifier               | Description
 |---                       |---
 | type                     | The type of object this json entry is describing. Setting this entry to 'armor' for example means the game will expect properties specific to armor in that entry. Also ties in with 'copy-from' (see below), if you want to inherit properties of another object, it must be of the same tipe.
-| copy-from                | The identifier of the item you wish to compy properties from. This allows you to make an exact copy of an item __of the same type__ and only provide entries that should change from the item you copied from. For example, if you created a small wood crate and wanted another but bigger, you can provide the ID of the small wood crate in the 'copy-from' property of the large wood crate so all the properties get copied, and adjust the volume and weight to what you need. This does not apply to all types, only types that have generic_factory implemented.
-| extends                  | Modders can add an "extends" field to their definition to append entries to a list instead of overriding the entire list.
-| delete                   | Modders can also add a "delete" field that removes elements from lists instead of overriding the entire list.
-| abstract                 | Creates an abstract item (an item that does not end up in the game and solely exists in the json to be copied-from. Use this _instead of_ 'id'. 
+| [copy-from](https://github.com/snipercup/Cataclysm-DDA/blob/DOC-move-JSON-inheritance-wiki-page-to-JSON_inheritance.md-and-update-it/doc/JSON_INHERITANCE.md)                | The identifier of the item you wish to copy properties from. This allows you to make an exact copy of an item __of the same type__ and only provide entries that should change from the item you copied from. 
+| [extends](https://github.com/snipercup/Cataclysm-DDA/blob/DOC-move-JSON-inheritance-wiki-page-to-JSON_inheritance.md-and-update-it/doc/JSON_INHERITANCE.md)                  | Modders can add an "extends" field to their definition to append entries to a list instead of overriding the entire list.
```

Comment:
```suggestion
| [extends](https://github.com/CleverRaven/Cataclysm-DDA/tree/master/doc/JSON_INHERITANCE.md)                  | Modders can add an "extends" field to their definition to append entries to a list instead of overriding the entire list.
```